controllers/articles.controllers.js

```diff
@@ -24,6 +24,19 @@ const getArticles = (req, res, next)=>{
     })
 }
 
+const getComments = (req, res, next)=>{
+    const {article_id}=req.params
+    fetchComments(article_id)
+    .then((data)=>{
```

Comment:
Small tip - if you were to call the response from fetchComments `comments` you could add it to the response object as a key only - `.send({comments})`